example.py

```diff
@@ -0,0 +1,68 @@
+import argparse
```

Comment:
./example.py is too generic. Should be examples/transport.py ?

Reply:
Ya.  This is a junk commit.  It would need a bit more cleaning up anyway.